### PR TITLE
[RDK-28151] Code cleanup

### PR DESCRIPTION
**Pull request by Koczek9**
Removed all references to LLAMA build, Hamiltron, NGDEV. Many references of SKY replaced with RDK.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -49,19 +49,17 @@ endif()
...
+set( RDK_PLATFORM "XI6" CACHE STRING "The target RDK platform, options are \"XI1\", \"XI6\" or \"DEV_VM\"" )
 string( TOUPPER "${RDK_PLATFORM}" RDK_PLATFORM )
 
 # If developer VM defined, configure platform to be Ubuntu VM machine, else configure set-top box
 if( RDK_PLATFORM STREQUAL "XI6" )
     add_definitions( -DRDK_PLATFORM=XI6 )
 elseif( RDK_PLATFORM STREQUAL "XI1" )
     add_definitions( -DRDK_PLATFORM=XI1 )
-elseif( RDK_PLATFORM STREQUAL "LLAMA" )
-    add_definitions( -DRDK_PLATFORM=LLAMA )
```
Comment (TeknoVenus) [inline CMakeLists.txt:64]: This codebase will need to support Llama builds - please revert these changes

File: README.md [hunk 2]
```diff
@@ -40,7 +40,6 @@ If the schema files in the `bundle/runtime-schemas` directory are changed, then
 * `-DRDK_PLATFORM=DEV_VM` - specifiy which platform Dobby is running on. Defaults to `XI6` if none specified. Valid options include
   * `XI6`
   * `XI1`
-  * `LLAMA`
```
Comment (TeknoVenus) [inline README.md:43]: Please revert.

File: bundle/lib/source/DobbyTemplate.cpp [hunk 3]
```diff
@@ -361,72 +358,6 @@ void DobbyTemplate::setTemplateEnvVars(const std::map<std::string, std::string>&
...
- *  For a list of model names refer to the following confluence page
- *
- *      https://www.stb.bskyb.com/confluence/display/2016/STB+HW+ID
- *
- *  To determine platform we check for the AI_PLATFORM_IDENT environment
- *  variable.  This should have been set by the user (or upstart script) that
- *  started this daemon.  If it's not present then we set both templates
- *  as empty and log an error.
- */
-void DobbyTemplate::setTemplatePlatformEnvVars()
```
Comment (TeknoVenus) [inline bundle/lib/source/DobbyTemplate.cpp:382]: Disagree about the need to remove this method - will need to remain for full compatibility with Sky builds

File: daemon/process/CMakeLists.txt [hunk 4]
```diff
@@ -115,8 +115,6 @@ if( RDK_PLATFORM STREQUAL "XI6" )
         set( DOBBY_SETTINGS_FILE "settings/dobby.xi6.json" )
 elseif( RDK_PLATFORM STREQUAL "XI1" )
         set( DOBBY_SETTINGS_FILE "settings/dobby.xi1.json" )
-elseif( RDK_PLATFORM STREQUAL "LLAMA" )
```
Comment (TeknoVenus) [inline daemon/process/CMakeLists.txt:118]: Please revert

File: daemon/process/settings/dobby.llama.json [hunk 5]
```diff
@@ -1,34 +0,0 @@
...
-
-  "network": {
-    "externalInterfaces": [ "eth0", "wlan0" ],
-    "addressRange": "100.64.11.0"
-  }
-
-}
-
-
-
```
Comment (TeknoVenus) [inline daemon/process/settings/dobby.llama.json:34]: Please revert

File: README.md [hunk 6]
```diff
@@ -50,7 +49,7 @@ If you with to develop Dobby further, detailed instructions on setting up a deve
 ## DobbyDaemon
 This is the main component of Dobby. This daemon is responsible for managing, controlling and creating containers. The daemon runs in the background and communicates over dbus. It connects on a few dbus addresses - one for admin, one for debugging and the other for Dobby commands.
 
-DobbyDaemon can be started as a systemd service at system boot (the systemd service file can be found at `daemon/process/sky-dobby.service`) or run manually on the command line by running:
+DobbyDaemon can be started as a systemd service at system boot (the systemd service file can be found at `daemon/process/rdk-dobby.service`) or run manually on the command line by running:
```
Comment (TeknoVenus) [inline README.md:52]: Prefer just `dobby.service` over `rdk-dobby.service`

**Review by TeknoVenus — CHANGES_REQUESTED** [review]
* `com.rdk` in various places should be changed to `org.rdk` 
* Llama support is still needed as we should be able to produce a functional llama build

File: daemon/process/settings/dobby.llama.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
+{
+  "dbus": {
+    "serviceName": "com.sky.dobby",
```
Comment (TeknoVenus) [inline daemon/process/settings/dobby.llama.json]: This is still `com.sky.dobby` (and other sky paths in this settings file need changing)

**Review by TeknoVenus — CHANGES_REQUESTED** [review]

File: daemon/process/settings/dobby.llama.json (hunk 7 above)
Comment (Koczek9) [reply daemon/process/settings/dobby.llama.json]: Nice catch, I missed this file as it was deleted when I was making all sky->rdk change, and now when it got reverted I missed it completely.

File: daemon/process/CMakeLists.txt [hunk 8]
```diff
@@ -126,20 +126,20 @@ endif()
...
         RENAME      dobby.json
-        DESTINATION /etc/sky/ )
+        DESTINATION /etc/rdk/ )
 
 
 
 # Install a systemd launch service config and enables it by default
 install(
-        FILES       sky-dobby.service
+        FILES       rdk-dobby.service
```
Comment (Anttikyll) [inline daemon/process/CMakeLists.txt:135]: As already pointed out by @TeknoVenus, `dobby.service` is preferred over `rdk-dobby.service`. The same can be found in quite a few other places as well.

As a note - this will require a change to the dobby recipe as well.

File: daemon/process/settings/dobby.dev_vm.json [hunk 9]
```diff
@@ -1,12 +1,12 @@
 {
     "dbus": {
-      "serviceName": "com.sky.dobby",
-      "objectPath": "/com/sky/dobby"
+      "serviceName": "org.rdk.dobby",
+      "objectPath": "/org/rdk/dobby"
```
Comment (Anttikyll) [inline daemon/process/settings/dobby.dev_vm.json:4]: We should reflect this change in the OCIContainers Thunder plugin as well, where we're still using `com.sky.dobby.test`.

**Comment by Anttikyll** [comment]
LGTM

**Review by TeknoVenus — APPROVED** [review]
LGTM
